src/js/devtools/App.jsx

```diff
@@ -17,9 +17,9 @@
 import React from 'react';
 import ExampleGraph from '../components/Graphs/exampleComponent';
 
-const App = () => (
+const App = (props) => (
   <div className="app">
-    <ExampleGraph />
+    <ExampleGraph {...props} />
```

Comment:
There is only one thing being passed via this object, so it's better to pass it directly